### PR TITLE
[Other]Right Rotate a list

### PR DESCRIPTION
**Pull request by nancyanand2807**
This is a python implementation to right rotate a list of n elements by k positions clockwise. The function rightRotate() returns the rotated list. The approach used in here is to traverse the elements in the first list one by one and then put the elements at required places in a second list.

### CODE REVIEW

File: rightrotate.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    for item in range(0, len(lists) - num):  
+        output_list.append(lists[item]) 
+          
+    return output_list 
+  
+# Driver Code 
+
+k = 4
+n = [7, 8, 9, 10, 11, 12] 
+
```
Comment (aneels3) [inline rightrotate.py]: hello @nancyanand2807 try to remove the unnecessary lines after each statement and do generalized code for this problem. your code is perfect for the given list and the given k.
make it a generic one such that it runs for every possible value of k and the user-defined list.

Comment (nancyanand2807) [reply rightrotate.py]: Sure. I will update it in some time.

Comment (aneels3) [reply rightrotate.py]: ok

Comment (nancyanand2807) [reply rightrotate.py]: Hey, I have updated the code as per the requirements. Kindly review it.

**Pull request merged**